src/lnp/session/noise/conduit.rs

```diff
@@ -0,0 +1,630 @@
+use super::{chacha, hkdf};
+use crate::lnp::session::transcoders::{Decrypt, Encrypt, Transcode};
+use amplify::Bipolar;
+
+use super::handshake::HandshakeError;
+use chacha20poly1305::aead::Error;
+use std::borrow::Borrow;
+
+pub type SymmetricKey = [u8; 32];
+
+/// Maximum Lightning message data length according to
+/// [BOLT-8](https://github.com/lightningnetwork/lightning-rfc/blob/v1.0/08-transport.md#lightning-message-specification)
+/// and [BOLT-1](https://github.com/lightningnetwork/lightning-rfc/blob/master/01-messaging.md#lightning-message-format):
+const LN_MAX_MSG_LEN: usize = ::std::u16::MAX as usize; // Must be equal to 65535
```

Comment:
We already have this constant here: https://github.com/LNP-BP/rust-lnpbp/blob/master/src/lnp/transport/mod.rs#L34-L46

It's better to use them, not create a new one

Reply:
I am juggling between `MAX_FRAME_PAYLOAD_SIZE` and `MAX_FRAME_SIZE`. It seems `MAX_FRAME_SIZE` would be a more appropriate constant to use here.